packages/manager/apps/web/client/app/domain/dns/sec/domain-dns-sec.controller.js

```diff
@@ -2,10 +2,19 @@ import get from 'lodash/get';
 
 angular.module('controllers').controller(
   'Domain.controllers.DnsSec',
+  /* @ngInject */
```

Comment:
We usually set it above the `constructor`, don't we ?

Reply:
Yes,  making the change, to keep consistency :)